### PR TITLE
Add ipreuse to parallel test

### PR DESCRIPTION
**Pull request by mboukhalfa**
The Idea of parallel tests is to run features on different VMs in the same time 
previously:
vm1 : run pivoting based tests
vm2 : run remedaition tests
vm3 : run live-iso

This PR edit stage of vm3 to run all the other feature tests excluding tests running in vm1 and vm2 so for now vm3 will run live-iso and ipreuse

### CODE REVIEW

File: jenkins/jobs/parallel_e2e_features_test.pipeline [hunk 1]
```diff
@@ -243,7 +243,8 @@ pipeline {
           environment {
             TEST_EXECUTER_VM_NAME = "${TEST_EXECUTER_VM_NAME}-live-iso"
             BUILD_TAG = "${env.BUILD_TAG}-live-iso"
-            GINKGO_FOCUS = "live-iso"
+            GINKGO_FOCUS = "features"
+            GINKGO_SKIP = "pivoting remediation"
```
Comment (lentzi90) [inline jenkins/jobs/parallel_e2e_features_test.pipeline:247]: The `stage` says this is e2e live iso test. I'm not understanding this. Is live-iso part of features? And does it work now to use both focus and skip? :thinking:

**Comment by kashifest** [comment]
> _ No description provided. _

why do we need this change? Whats the idea?

File: jenkins/jobs/parallel_e2e_features_test.pipeline (hunk 1 above)
Comment (mboukhalfa) [reply jenkins/jobs/parallel_e2e_features_test.pipeline:247]: Thanks for spotting that ! the third stage should run feature tests that are not part of pivoting-based feature nor remediation features for instance live-iso and IPReuse that are feature tests but not part of pivoting-based and remediation.
I expect it to work with both first focus on all features and exclude skipped once so this stage should run all the features tests excluding pivoting based and remediation

Comment (mboukhalfa) [reply jenkins/jobs/parallel_e2e_features_test.pipeline:247]: fixed

**Comment by mboukhalfa** [comment]
> > _ No description provided. _
> 
> why do we need this change? Whats the idea?

Description added

File: jenkins/jobs/parallel_e2e_features_test.pipeline [hunk 2]
```diff
@@ -111,7 +111,7 @@ pipeline {
           environment {
             TEST_EXECUTER_VM_NAME = "${TEST_EXECUTER_VM_NAME}-pivoting-based"
             BUILD_TAG = "${env.BUILD_TAG}-pivoting-based"
-            GINKGO_FOCUS = "ephemeral"
+            GINKGO_FOCUS = "pivoting"
```
Comment (kashifest) [inline jenkins/jobs/parallel_e2e_features_test.pipeline:114]: so ephemeral was doing which tests ?

File: jenkins/jobs/parallel_e2e_features_test.pipeline (hunk 1 above)
Comment (lentzi90) [reply jenkins/jobs/parallel_e2e_features_test.pipeline:247]: Thanks for explaining, sounds good! The `BUILD_TAG` should also be changed though (line 245)

**Review by kashifest — APPROVED** [review]

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Comment by mboukhalfa** [comment]
Sorry I did not know that GINKGO_SKIP propagating was removed here https://github.com/metal3-io/project-infra/pull/523 ! 

/parallel-test-ubuntu-e2e-feature-main

**Comment by mboukhalfa** [comment]
infra issue 
/parallel-test-ubuntu-e2e-feature-main

**Comment by mboukhalfa** [comment]
@lentzi90 Can you re LGTM !

File: jenkins/scripts/integration_test.sh
Comment (lentzi90) [inline jenkins/scripts/integration_test.sh:1]: All these formatting changes for a single line change :roll_eyes: 
Would have been nice to just save without formatting

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Review by kashifest — APPROVED** [review]

**Pull request merged**